src/app/pages/common/ix-forms/components/ix-input/ix-input.component.ts

```diff
@@ -33,13 +42,32 @@ export class IxInputComponent implements ControlValueAccessor {
     this.controlDirective.valueAccessor = this;
   }
 
-  writeValue(value: string): void {
-    this.value = value;
+  writeValue(value: string | number): void {
+    let parsed = value;
+    let formatted = value;
+    if (value) {
+      if (this.parse) {
+        parsed = this.parse(value);
+        /** This flag exists because when default value is used, onChange is not registered yet
+         * So parsed value isn't immediately updated. Furthermore, if the user submits the form with
+         * default value, the submitted value isn't parsed in that case either.
+         */
+        this.shouldUpdateValueAsap = true;
```

Comment:
I still don't think we need this. 
It's easier to just require everyone to use proper numeric values in default values or use a service, i.e.:
```
memory_size: [this.ixFormatter.memorySizeParsing('2 mib')],
```
And I would imagine that data is coming in numbers from middleware anyway.